Source/WebKit/UIProcess/API/APITargetedElementRequest.h

```diff
@@ -36,7 +36,6 @@ namespace API {
 
 class TargetedElementRequest final : public ObjectImpl<Object::Type::TargetedElementRequest> {
 public:
-
```

Comment:
Whitespace

Reply:
Ah, yeah — I'd intentionally removed the extraneous newline here (hopefully okay to come along for the ride 😅)